framework/base/Behavior.php

```diff
@@ -86,7 +86,15 @@ public function detach()
     {
         if ($this->owner) {
             foreach ($this->events() as $event => $handler) {
-                $this->owner->off($event, is_string($handler) ? [$this, $handler] : $handler);
+                if (is_string($handler)) {
+                    $sParam = [$this, $handler];
```

Comment:
What does `sParam` mean?

Reply:
`second param`

Reply:
Second parameter of the method is `$handler`. I'd suggest naming the variable like that instead of "sParam".